docs/direct_mig_requirements.md

```diff
@@ -0,0 +1,26 @@
+# Direct Migration Requirements
+
+Direct Migration is available as a feature from Migration Toolkit for Containers (MTC) release 1.4.0. There are two parts of the Direct Migration - Direct 
+Volume Migration and Direct Image Migration. Direct Migration enables the migration of persistent volumes and internal
+images directly from the source cluster to the destination cluster without an intermediary replication repository (object storage)
+There are some requirements which are to be satisfied before you can use the Direct Migration feature of MTC.
+
+1. The clusters (source as well as destination) involved in the migration should expose their respective internal registries
+for external traffic.
+
+2. The remote source and destination clusters should be able to communicate via the OpenShift routes on port 443.
```

Comment:
Does the user have to explicitly allow this port? Or is 443 allowed by default?

Reply:
443 port is used by default for TLS enabled route (https).